src/bosh-director/lib/bosh/director/task_db_writer.rb

```diff
@@ -0,0 +1,15 @@
+module Bosh::Director
+  class TaskDBWriter
+    def initialize(column, task)
+      @column_name = column
+      @task = task
+      @transactor = Transactor.new
+    end
+
+    def write(text)
+      @transactor.retryable_transaction(Bosh::Director::Config.db) do
+        @task.update({@column_name => "#{@task[@column_name]}#{text}"})
```

Comment:
@DennisDenuto Thank you for the review. Could you please provide any sample of the sequel concat for model objects? So far I found that `Sequel.x` works only for dataset actions. And in case if it is the dataset action, to update the model object I need to use `task.refresh`, so no reduction of interactions with db.